### ISSUE TITLE
jsx-indent Multiline parentheses remove indentation warnings for tags

### ISSUE THREAD

**Issue by doque** [yannickcr/eslint-plugin-react#500]
Eslint v2.2.0

The following code does not throw an error/warning for the incorrect indentation of the `<div>` element.
When removing the surrounding brackets, the warning appears.

```
import React from 'react';

const StatelessComponent = () => {
    if (new Date() % 2) {
        return (
<div>Hello</div>
        );
    }
    return null;
};

export default StatelessComponent;
```

Note - indentation for props (when put in a new line) works as expected and is not affected by this.
